dataplane/src/main.rs

```diff
@@ -102,7 +102,7 @@ fn init_devices(eal: &Eal) -> Vec<Dev> {
         .collect()
 }
 
-fn start_rte_workers(devices: &Vec<Dev>) {
+fn start_rte_workers(devices: &[Dev]) {
```

Comment:
non blocking note:

This change is perfectly fine here (and is likely best in this case).

In the more general case of a function which needs to accept a borrowed, fixed length, random access iterator, I generally write this:

```rust
fn start_rte_workers<T: AsRef<Dev>>(devices: T) {
    let devices = devices.as_ref();
    /* whatever */
}
```

That API can be more friendly (especially in the case of functions which accept any variant of `str`).

In this case this function is one-and-done so no need for the generic.

Reply:
I suspect the interface manager is going to change some of this in main anyway so I vote to leave as is here.

Reply:
This got pushed in a cleanup PR, so is now moot here.